x/epoching/keeper/keeper.go

```diff
@@ -20,7 +19,6 @@ type (
 		hooks        types.EpochingHooks
 		bk           types.BankKeeper
 		stk          types.StakingKeeper
```

Comment:
Does it make sense to only pass `stk` in the `NewKeeper` function (without passing `stkMsgServer`), and instead use `stakingkeeper.NewMsgServerImpl(stakingKeeper)` to construct the `stkMsgServer` in tke `NewKeeper` function?

Reply:
it could be, but then in the epoching keeper I would expect to receive the pointer to the staking module instead of the expected keeper staking interface